app/screens/channel/channel_post_list/index.ts

```diff
@@ -47,4 +48,4 @@ const enhanced = withObservables(['channelId', 'forceQueryAfterAppState'], ({dat
     };
 });
 
-export default withDatabase(enhanced(ChannelPostList));
```

Comment:
Similar to the navigation header, the ChannelPostList was completely re-rendered when the SafeAreaProvider gets initialized. This takes away around 304ms (since this is on my test channel, and depends on the number and shape of posts, the value may vary).